backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -255,7 +255,7 @@ const parseTemplateMessageWebhook = async (
       }
       const message = (govsgMessage ?? govsgOp) as GovsgMessage
       const experimentalData = await experimentService.getUserExperimentalData(
-        message.campaign.userId
```

Comment:
Could I ask how you tested this?

Reply:
Sorry, I don't understand your qn...

If you mean how I found out, it's through Sentry.

If you mean how I ensured this works, I read the schema and got this to work.